docusaurus/docs/Android/03-ui/04-components/03-message-list.md

```diff
@@ -24,62 +77,113 @@
 | --- | --- |
 |![Message_options_in light mode](../../assets/message_options_light.png)|![Message options in dark mode](../../assets/message_options_dark.png)|
 
-Most of the actions work out of the box but you can change their behavior using different listeners and handlers:
```

Comment:
I cannot comment above this line but I would replace this sentence ```MessageListView comes with a set of actions out of the box that are available by long-pressing a message. There are multiple available actions:``` with ```MessageListView comes with a set of actions available out of the box by long-pressing a message:```. Moreover some of the listed actions are hidden by default and we also allow to Unmute users